src/scripts/components/embed/Embed.jsx

```diff
@@ -37,20 +37,50 @@ class Embed extends Component {
   render() {
     const { audio } = this.state;
 
+    const waveSurferOptions = {
+      backend: "MediaElement",
+      barWidth: 1,
+      cursorWidth: 0,
+      height: 150,
+      normalize: true,
+      progressColor: audio.progressColor ? audio.progressColor : "#0fb3cc",
+      responsive: true,
+      waveColor: audio.waveColor ? audio.waveColor : "#a2e0e3"
+    };
+
     return (
       <div id="embed">
         <h3>{audio.title}</h3>
         <p>{audio.contributors}</p>
         {audio.image && <img id="embed__image" src={audio.image} />}
-        <Wavesurfer
-          audioFile={`http://localhost:3000/${audio.url}`}
-          onPosChange={this.handlePosChange}
-          playing={this.state.playing}
-        />
+        {audio.url && (
+          <div
+            id="embed__audio-player"
+            style={{
+              border: "1px solid #e5e5e5",
```

Comment:
Would this inline style (for the border) work better in a separate css file?

Reply:
You're right! I moved it to a separate css file